src/components/tasks/card/index.tsx

```diff
@@ -240,17 +259,30 @@ const Card: FC<Props> = ({
         </span>
         <span>
           <span className={classNames.cardSpecialFont}>Assignee:</span>
+          { IsUserAuthorized && isAssigneeDropdownOpen && shouldEdit
+            ? (
+             <AssigneeDropdownMenu
+                cardDetails={cardDetails}
+                updateAssignee={updateAssignee}
+                setisAssigneeDropdownOpen={setisAssigneeDropdownOpen}
+              /> )
+              : cardDetails.assignee ? (
           <span
-            className={classNames.cardStrongFont}
-            contentEditable={shouldEdit}
-            onKeyPress={(e) => handleChange(e, 'assignee')}
+            className={`${classNames.cardStrongFont} ${shouldEdit && classNames.cursorPointer}`}
             role="button"
             tabIndex={0}
-          >
+            onClick={() => setisAssigneeDropdownOpen(!isAssigneeDropdownOpen)}
+          > 
             {cardDetails.assignee}
           </span>
+              )
+            : shouldEdit && <span 
+            onClick={() => setisAssigneeDropdownOpen(!isAssigneeDropdownOpen)}
+            className={`${classNames.cardStrongFont} ${classNames.cursorPointer}`} role="button"
+            tabIndex={0}>Unassigned</span>
+          }
           <span
-            className={classNames.contributorImage}
+            className={`${classNames.contributorImage}`}
```

Comment:
Was there an error created for this?

Reply:
no i think i was adding multiple class names before the enhancement , I missed to update them